CareKitStore/CareKitStore/Structs/OCKOutcomeValue.swift

```diff
@@ -58,41 +58,41 @@ public struct OCKOutcomeValue: Codable, Equatable, OCKObjectCompatible, CustomSt
         case
         kind, units, uuid, value, type, index,
         createdDate, updatedDate, schemaVersion, tags,
-        group, remoteID, userInfo, source, timezone
```

Comment:
Here's the key, I didn't see it like this for any other entity.

Reply:
Any idea why unit tests aren't running for this PR?

Reply:
I'm not sure. I just ran GitHub Actions on a different repo, so I don't think it's down

Reply:
I know why, the change to "main" isn't in the actions file, I can make the change

Reply:
Ah, that'll do it! Thanks for tweaking that!